stacks-signer/Cargo.toml

```diff
@@ -31,6 +31,7 @@ reqwest = { version = "0.11.22", default-features = false, features = ["blocking
 serde = "1"
 serde_derive = "1"
 serde_stacker = "0.1"
+sha2 = "0.10"
```

Comment:
Can you please instead use the hashing algorithms in `stacks_common`?  They pull in `sha2` as a transitive dependency as it is.

Reply:
Sure thing, I'll update.

Reply:
Done.